build_container/build_container_ubuntu.sh

```diff
@@ -86,24 +75,31 @@ case $ARCH in
     'ppc64le' )
         LLVM_DISTRO=powerpc64le-linux-ubuntu-18.04
         LLVM_SHA256SUM=2d504c4920885c86b306358846178bc2232dfac83b47c3b1d05861a8162980e6
+        apt-get install -y --no-install-recommends \
+        libtinfo5 # LLVM dependencies on Ubuntu 20.04
         ;;
     'x86_64' )
         LLVM_DISTRO=x86_64-linux-gnu-ubuntu-18.04
         LLVM_SHA256SUM=61582215dafafb7b576ea30cc136be92c877ba1f1c31ddbbd372d6d65622fef5
+        apt-get install -y --no-install-recommends \
+        google-cloud-sdk \
```

Comment:
why do we need google-cloud-sdk here? you can add this only if x86_64 and aarch64 to PACKAGES in above.

Reply:
I'm not sure whether there is a more elegant solution but I'll append the google-cloud-sdk if ARCH is x86 or arm ok?
Will look like this:

```bash
PACKAGES=(
    [...]
)

if [[ "${ARCH}" == "x86_64" || "${ARCH}" == "aarch64" ]]; then
  PACKAGES+=("google-cloud-sdk")
fi

```

I'll also add libtinfo5 in PACKAGES, then we can remove the complete `apt-get install ...` in the LLVM part.

Would that be fine with you?